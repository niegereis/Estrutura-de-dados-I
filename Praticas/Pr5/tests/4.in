10
Enrico 1.8 21.2.1383
Melissa 7.8 21.2.9034
Livia 7.2 18.1.9207
Rafael 6.8 25.2.9825
PedroMiguel 9.9 17.2.0480
Emilly 7.7 23.2.8985
ArthurGabriel 6.8 13.1.8529
Laura 5.1 25.2.0638
Vinicius 2.0 12.2.0385
Ayla 3.1 21.2.9252
9 8