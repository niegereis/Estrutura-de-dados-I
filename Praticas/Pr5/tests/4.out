21.2.1383 1.8
21.2.9034 7.8
18.1.9207 7.2
25.2.9825 6.8
17.2.0480 9.9
23.2.8985 7.7
13.1.8529 6.8
12.2.0385 2.0
25.2.0638 5.1
21.2.9252 3.1
